crates/stdarch-gen-arm/spec/neon/arm_shared.spec.yml

```diff
@@ -14184,8 +13816,7 @@ intrinsics:
     visibility: private
     doc: "Store multiple single-element structures from one, two, three, or four registers."
     arguments: ["addr: {type[1]}", "val: {neon_type[2]}", "align: {type[3]}"]
-    safety:
-      unsafe: [neon]
+    safety: safe
```

Comment:
This should be marked as unsafe as it operates on memory; for example [vst1_s8](https://developer.arm.com/architectures/instruction-sets/intrinsics/vst1_s8):

> Store multiple single-element structures from one, two, three, or four registers. This instruction stores elements to memory from one, two, three, or four SIMD&FP registers, without interleaving. Every element of each register is stored.